src/Operation/BulkWrite.php

```diff
@@ -91,6 +91,14 @@ class BulkWrite implements Executable
      *  * upsert (boolean): When true, a new document is created if no document
      *    matches the query. The default is false.
      *
+     * Supported options for replaceOne and updateOne operations:
+     *
+     *  * sort (document): Determines which document the operation modifies if
+     *    the query selects multiple documents.
+     *
+     *    This is not supported for server versions < 8.0 and will result in an
+     *    exception at execution time if used.
```

Comment:
The meaning of "exception at execution" wasn't clear to me on first reading. I would have understood better with "runtime exception" or "server error". But we can keep that as it's consistent with the rest of the project.

Reply:
Aye, this language is used throughout the API documentation (both actual docs and these comments) to distinguish between exceptions raised during construction and during a call to `execute()`. I wouldn't object to changing it, but it should be done across the board if so.